editor/src/components/canvas/canvas-strategies/strategies/groups.spec.browser2.tsx

```diff
@@ -59,6 +60,22 @@ async function dragByPixels(
   })
 }
 
+async function changeInspectorNumberControl(
```

Comment:
Probably worth putting this in a test utils file somewhere.

Reply:
done!